sphinx/parsers.py

```diff
@@ -55,15 +53,16 @@ def set_application(self, app):
         self.info = app.info
 
 
-class RSTParser(docutils.parsers.rst.Parser):
+class RSTParser(docutils.parsers.rst.Parser, Parser):
```

Comment:
now inheriting from Parser is not needed, because we don't need access to `app`. Should I delete that added inheritance ?